jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/AdvanceRuntimeDataServiceImplTest.java

```diff
@@ -352,7 +349,7 @@ public void testQueryTaskNotEqualsByVariables() {
     @Test
     public void testQueryTaskByVariablesWithOwners() {
 
-        List<String> potOwners = Collections.singletonList("katy");
+        QueryParam potOwners = new QueryParam(null, "ALL", Collections.singletonList("katy"));
```

Comment:
QueryParam as others please. QueryParam.all ...QueryParam.any (Static methods)

Reply:
Sure, that's much better, thanks. Updated tests accordingly